pages/index.tsx

```diff
@@ -1,64 +1,77 @@
-import { ChangeEvent, useEffect, useState } from "react";
-import { CrossCircleSVG, Spinner, Tag } from "@ensdomains/thorin";
+import { ChangeEvent, useState } from "react";
+import { CrossCircleSVG } from "@ensdomains/thorin";
 import { HomepageBg } from "@/components/01-atoms";
 import { isNameAvailable } from "@/lib/utils/blockchain-txs";
 import { ENSName, buildENSName } from "@namehash/ens-utils";
 import { DebounceInput } from "react-debounce-input";
 import { useRouter } from "next/router";
 import { normalize } from "viem/ens";
 import Link from "next/link";
+import { EnsDomainStatus } from "@/types/ensDomainStatus";
+import { domainWithEth } from "@/lib/utils/formats";
+import { EnsDomainStatusComponents } from "@/components/02-molecules";
 
 export default function Home() {
   const router = useRouter();
   const [domain, setDomain] = useState("");
-  const [isAvailable, setIsAvailable] = useState<boolean | null>(null);
-  const [domainError, setDomainError] = useState(false);
+  const [domainStatus, setDomainStatus] = useState<EnsDomainStatus>(
+    EnsDomainStatus.Available
```

Comment:
Since the EnsDomainStatus "available" is a non-blocker and lets the user go to the next step, I think that the default status should be a blocker status, like "Invalid".

Reply:
Good catch!